pysatNASA/instruments/methods/_cdf.py

```diff
@@ -91,8 +91,13 @@ def __init__(self, filename,
         self.meta = {}
         self._dependencies = {}
 
-        self._variable_names = (self._cdf_info['rVariables']
-                                + self._cdf_info['zVariables'])
+        try:
+            self._variable_names = (self._cdf_info.rVariables
+                                    + self._cdf_info.zVariables)
+        except AttributeError:
+            # cdflib < 1.0 stores info as a dict
+            self._variable_names = (self._cdf_info['rVariables']
+                                    + self._cdf_info['zVariables'])
```

Comment:
```suggestion
        if hasattr(self._cdf_info, 'rVariables') and hasattr(self._cdf_info, 'zVariables'):
            self._variable_names = (self._cdf_info.rVariables
                                    + self._cdf_info.zVariables)
        else:
            # cdflib < 1.0 stores info as a dict
            self._variable_names = (self._cdf_info['rVariables']
                                    + self._cdf_info['zVariables'])
```

Another stylistic possibility to consider for the code here and throughout this file.

Reply:
I like this.